velox/core/QueryCtx.h

```diff
@@ -207,8 +220,8 @@ class ExecCtx : public Context {
 
  private:
   // Pool for all Buffers for this thread
-  memory::MemoryPool* pool_;
-  QueryCtx* queryCtx_;
+  memory::MemoryPool* FOLLY_NONNULL pool_;
```

Comment:
It would be nice to extract adding FOLLY_NONNULL to existing code into a separate PR. This would make this PR shorter and allow reviewers to focus on material changes.